src/main/kotlin/uk/gov/justice/digital/hmpps/welcometoprison/model/prison/temporaryabsences/ConfirmTemporaryAbsenceResponse.kt

```diff
@@ -4,5 +4,6 @@ import com.fasterxml.jackson.annotation.JsonInclude
 
 @JsonInclude(JsonInclude.Include.NON_NULL)
 data class ConfirmTemporaryAbsenceResponse(
-  val prisonerNumber: String,
+  val prisonNumber: String,
```

Comment:
these response objects should have swagger annotations

Reply:
done